build-system/compile/OWNERS

```diff
@@ -20,6 +20,7 @@
         {name: 'ampproject/wg-components'},
         {name: 'ampproject/wg-infra'},
         {name: 'ampproject/wg-performance'},
+        {name: 'ampproject/wg-stories'},
```

Comment:
Retroactive, but at this point would it make sense to just set it as `{name: '*'}`?